error_handling/global-error-handler.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Global Error Handler
+parent: Error Handling
+---
+
+```js
+// app/javascript/packs/application.js
+import { Application } from "stimulus"
+import { definitionsFromContext } from "stimulus/webpack-helpers"
+
+const application = Application.start()
+const context = require.context("./controllers", true, /\.js$/)
+application.load(definitionsFromContext(context))
+
+
+// configure Sentry to log errors
+const defaultErrorHandler = application.handleError
```

Comment:
@adrienpoly so you “memorize” the `defaultErrorHandler` here to do a kind of `super` call in L20 and then redefine the `application.handleError` to use the new `sentryErrorHandler` ?

Reply:
yes exactly the default `application.handleError` function of Stimulus will just basically console.log the error in a nice format. 
Here what I am doing is first reporting this error to a tracking system (Sentry here) then preserving the usual console.log error of Stimulus by calling the `memorized` function